modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/action/executor/SplitCommerceOrderByCatalogObjectActionExecutorImpl.java

```diff
@@ -219,94 +221,113 @@ private void _createSupplierOrders(
 		RoundingMode roundingMode = RoundingMode.valueOf(
 			commerceCurrency.getRoundingMode());
 
-		for (List<CommerceOrderItem> commerceOrderItemList :
-				commerceOrderItemMap.values()) {
+		commerceOrderItemMap.forEach(
+			(commerceCatalog, commerceOrderItems) -> {
+				CommerceOrder supplierCommerceOrder =
+					customerCommerceOrder.cloneWithOriginalValues();
 
-			CommerceOrder supplierCommerceOrder =
-				customerCommerceOrder.cloneWithOriginalValues();
+				supplierCommerceOrder.setUuid(_portalUUID.generate());
+				supplierCommerceOrder.setExternalReferenceCode(
+					_portalUUID.generate());
 
```

Comment:
we said to add the manually adjusted to the supplier order also

Reply:
Yes, it was done to the line 269

Reply:
done to the line 291